files/ko/glossary/bootstrap/index.html

```diff
@@ -0,0 +1,22 @@
+---
+title: 부트스트랩
+slug: Glossary/Bootstrap
+tags:
+  - 부트스트랩
+  - bootstrap
+  - Glossary
+  - framework
+  - css
+translation_of: Glossary/Bootstrap
+---
+<p>Bootstrap은 반응형 웹사이트를 빠르게 만들기 위한 무료 오픈소스 {{Glossary("HTML")}}, CSS 및 {{ Glossary ( " JavaScript " ) }} 의 프레임워크 입니다.</p>
+
+<p>Bootstrap은 당초 Twitter Blueprint라고 불렸으며, <a href="https://twitter.com/"> Twitter</a> 개발자에 의해 개발되었습니다. 반응형 디자인을 지원하며 그대로 사용하거나 필요에 따라 자신의 코드에 맞추어 사용할 수 있는 사전 정의된 디자인 템플릿이 있습니다. 부트스트랩은 모든 최신 브라우저와 {{glossary("Microsoft Internet Explorer", "Internet Explorer")}} 의 최근 버전과 호환되기 때문에 타 브라우저와의 호환성에 대해 걱정하지 않으셔도 됩니다.</p>
```

Comment:
```suggestion
<p>Bootstrap은 당초 Twitter Blueprint라고 불렸으며, <a href="https://twitter.com/"> Twitter</a> 개발자에 의해 개발되었습니다. 반응형 디자인을 지원하며 그대로 사용하거나 필요에 따라 자신의 코드에 맞추어 사용할 수 있는 사전 정의된 디자인 템플릿이 있습니다. 부트스트랩은 모든 모던 브라우저와 {{glossary("Microsoft Internet Explorer", "Internet Explorer")}} 의 최근 버전과 호환되기 때문에 타 브라우저와의 호환성에 대해 걱정하지 않으셔도 됩니다.</p>
```

Modern Browsers를 최신 브라우저라고 해석하기엔 오역이 될 가능성이 보입니다. 이를 표음대로 적는게 좋지 않을까 싶습니다.